string text
int64 speaker_id
---